app/medInria/medHomepageArea.cpp

```diff
@@ -128,13 +128,19 @@ medHomepageArea::medHomepageArea ( QWidget * parent ) : QWidget ( parent ), d (
     medLogo = medLogo.scaled(350, 150, Qt::IgnoreAspectRatio, Qt::SmoothTransformation);
     medInriaLabel->setPixmap ( medLogo );
 
+    QDate expiryDate = QDate::fromString(QString(MEDINRIA_BUILD_DATE), "dd_MM_yyyy").addYears(1);
     QTextEdit * textEdit = new QTextEdit(this);
     textEdit->setHtml ( QString::fromUtf8("<b>MUSIC</b> is a software developed in collaboration with "
                                           "the IHU LIRYC in order to propose functionalities "
                                           "dedicated to cardiac interventional planning and "
                                           "guidance, based on the medInria software platform."
                                           "<br/><br/>"
-                                          "<b>MUSIC</b> is proprietary software, copyright (c) 2014-2015, IHU Liryc, Université de Bordeaux and Inria." ));
+                                          "<b>MUSIC</b> is proprietary software, copyright (c) 2014-2015, IHU Liryc, Université de Bordeaux and Inria."
+                                          "<br/><br/>"
+                                          " <font color = 'red'><b>This MUSIC copy will expire on ")
+                        + expiryDate.toString("d MMMM yyyy")
```

Comment:
I know the US uses dates like 01/12/16 instead of 12/01/16 (and I find it ridiculous too).
I wasn't talking about numerical only dates, but when the month is in word form:
January 12, 2016 instead of 12 January, 2016
but apparently even that is only US (I prefer the US version though, for once).